src/Compilers/CSharp/Test/Semantic/IOperation/ioperationtests_isymbolinitializer.cs

```diff
@@ -0,0 +1,270 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Still incorrect on the file system though. I see the right thing in VS (probably because that's the case specified in the csproj)